engine/apps/metrics_exporter/helpers.py

```diff
@@ -23,10 +23,14 @@
 
 def get_organization_ids():
     """Try getting organizations ids from cache, otherwise get from db and save values in cache"""
-    Organization = apps.get_model("user_management", "Organization")
+    AlertReceiveChannel = apps.get_model("alerts", "AlertReceiveChannel")
     organizations_ids = cache.get(METRICS_ORGANIZATIONS_IDS, [])
     if not organizations_ids:
-        organizations_ids = Organization.objects.all().values_list("id", flat=True)
+        organizations_ids = (
+            AlertReceiveChannel.objects.filter(organization__deleted_at__isnull=True)
```

Comment:
Since we are using the same query in 2 places, maybe we can move this to a separate helper?